### ISSUE TITLE
hdiutil not available on linux

### ISSUE THREAD

**Issue by mineek** [NightstarSakura/sunst0rm-Linux#1]
Hello!

hdiutil isn't available on linux, so mounting the dmg will fail with command not found.
This is an issue i couldnt get around myself aswell.
So fyi it doesnt work atm.

**Comment by NightstarSakura** [comment]
Thanks for feedback, this error we tring to fix.

**Comment by NightstarSakura** [comment]
hdiutil can use [darling](https://github/darlinghq/darling) to use, I will push commit later.

**Comment by NightstarSakura** [comment]
I fix hdiutil can't use in Linux, commit (here)[https://github.com/NightstarSakura/sunst0rm-Linux/commit/1531b0]

**Comment by mineek** [comment]
Nice! I will test it when i get home.

**Comment by NightstarSakura** [comment]
Of course! That my pleasure, but i just make it can success to patch ramdisk, because my iPhone 6S Plus can't enter pwndfu mode, so i can't test more, every problem please feedback to me, i will solve it as fast.

**Comment by mineek** [comment]
So i've tried the changes but when it copy's it says cp: read only file system. Soooo does hdiutil on linux only mount read only?

**Comment by NightstarSakura** [comment]
Can you tell me where said this error(like what text show close to the error)
I need to locate where this error occurs

**Comment by mineek** [comment]
When copying back the patcher asr into the ramdisk.

**Comment by mineek** [comment]
line 72 in sunst0rm and line 89
